lib/bundler/audit/database.rb

```diff
@@ -91,15 +94,29 @@ def self.path
       #
       # @since 0.3.0
       #
-      def self.update!
+      def self.update!(quiet)
         if File.directory?(USER_PATH)
           if File.directory?(File.join(USER_PATH, ".git"))
             Dir.chdir(USER_PATH) do
-              system 'git', 'pull', 'origin', 'master'
+              command = [
+                'git',
+                'pull',
+                quiet ? '--quiet' : nil,
+                'origin',
+                'master'
+              ].compact
+              system *command
             end
           end
         else
-          system 'git', 'clone', URL, USER_PATH
+          command = [
+            'git',
+            'clone',
+            quiet ? '--quiet' : nil,
+            URL,
+            USER_PATH
+          ].compact
+          system *command
```

Comment:
The "appending arguments" style is fine by me. I will make that change.

The "splatting nil" style is new to me, and I'm not sure most people would be familiar with it.